Firestore/Example/Podfile

```diff
@@ -27,6 +32,12 @@ target 'Firestore_Example_iOS' do
     pod 'ProtobufCpp', :podspec => 'ProtobufCpp.podspec'
   end
 
+  target 'Firestore_Benchmarks_iOS' do
+    inherit! :search_paths
+
+    pod 'GoogleBenchmark', :podspec => 'Tests/GoogleBenchmark/GoogleBenchmark.podspec'
```

Comment:
Please put this podspec just directly within `Firestore/Example` (like ProtobufCpp and GoogleTest above).

Reply:
Done.